datajunction-server/datajunction_server/alembic/env.py

```diff
@@ -84,21 +88,21 @@ def run_migrations_online():
     and associate a connection with the context.
 
     """
-    x_args = context.get_x_argument(as_dictionary=True)
+    x_args = alembic.context.get_x_argument(as_dictionary=True)
     connectable = create_engine(x_args.get("uri") or DEFAULT_URI)
 
     with connectable.connect() as connection:
-        context.configure(
+        alembic.context.configure(
             connection=connection,
             target_metadata=target_metadata,
             render_as_batch=True,
         )
 
-        with context.begin_transaction():
-            context.run_migrations()
+        with alembic.context.begin_transaction():
+            alembic.context.run_migrations()
 
 
-if context.is_offline_mode():
+if alembic.context.is_offline_mode():
```

Comment:
How does this top level if statement work now that it's part of the library? Whenever this is imported wouldn't that mean the migration runs? Is that what we want? What I'm trying to think through is what happens when you're deploying this to a cluster and the containers are all trying to migrate on startup at the same time.

Reply:
Hmm... that's a good point.

Reply:
This is never imported though, as far as I can tell. It's just used for configuring alembic, but we don't use it anywhere in the actual server code.